django-verdant/rca/static/rca/css/components/slider.less

```diff
@@ -0,0 +1,115 @@
+.slider {
+    &__button {
+        //reset global button styles
+        line-height: 0;
+        padding: 0;
+        display: block;
+        text-transform: none;
+        background-color: transparent;
+        border: 0;
+        font-size: 0;
+
+        &:hover {
+            background-color: transparent;
+        }
+
+        // apply button styles
+        position: absolute;
+        background-image: none;
+        width: 27px;
+        height: 49px;
+        z-index: 1;
+
+        &--next {
+            right: 16px;
+            top: calc(~"50% - 31px");
+
+            @media @media-desktop-regular {
+                top: 30%;
```

Comment:
The positioning of the arrows will vary according the block, and the events one has a different size at desktop to mobile - so not all of this is totally reusable. 

Also the previous button being hidden is going to be ditched as that was the old mobile design for the showcase - the new one shows both buttons.